transport_test.go

```diff
@@ -251,7 +250,7 @@ func TestRefreshTokenWithParameters(t *testing.T) {
 			if err != nil {
 				return nil, fmt.Errorf("error converting token into io.ReadWriter: %+v", err)
 			}
-			tokenBody := ioutil.NopCloser(tokenReadWriter)
+			tokenBody := os.NopCloser(tokenReadWriter)
```

Comment:
```suggestion
			tokenBody := io.NopCloser(tokenReadWriter)
```

here + elsewhere in the file